### ISSUE TITLE
Method SubCommands not working

### ISSUE THREAD

**Issue by sgaamuwa** [kakawait/picocli-spring-boot-starter#9]
In Picocli 3.6 there is the option to have the methods annotated with the @Command and have them listed as subcommands in that particular class. This doesn't seem to be picked up. Can subcommands only be achieved through nesting of classes?

**Comment by kakawait** [comment]
@sgaamuwa Yes current starter does not yet integrate new feature from Picocli >= 3.

I'll work with @remkop to integrate that thing asap
